maxim/logger/portkey/portkey.py

```
@@ -0,0 +1,200 @@
+        """Delegate all other attributes to the underlying chat client."""
+        return getattr(self._client.chat, name)
+
+
+class MaximPortkeyChatCompletions:
+    """Maxim instrumenter for Portkey chat completions."""
+
+    def __init__(self, client: Portkey, logger: Logger):
+        self._client = client
+        self._logger = logger
+
+    def create(self, *args, **kwargs):
```

Ruff (0.11.9)

Missing return type annotation for public function `create`

(ANN201)